src/views/Landing.js

```diff
@@ -34,6 +34,20 @@ const Landing = (props) =>
         </P>
 
         {
+          <Row>
+            <Col>
+              <Button
+                prop-type={'link'}
+                prop-size={'md'}
+                onClick={ () => props.pushRoute(ROUTE_NAMES.VIEW_SHIP) }
+              >
+                { 'View a point  →' }
```

Comment:
This should probably use the same size/styling as the "Unlock a Wallet" above? Looks kinda off right now.